CreatePackageDescr.pm

```diff
@@ -132,6 +132,9 @@ sub package_snippet($) {
         next
           if ( $name eq "installation-images-debuginfodeps-openSUSE"
             && $prv =~ m/debuginfo.build/ );
+        next
+          if ( $name eq "installation-images-debuginfodeps-Kubic"
```

Comment:
Perhaps we should switch to a `.*` for that future `-DimStar` target?

Still rather ugly this needs to be done, but better than whitelisting whole package.

Reply:
.* sounds reasonable to me; I'm sure any future/potential i-i-debuginfodeps-* will have debuginfo.build deps, which we won't be able to handle.. let me change that